### ISSUE TITLE
Booru.org subdomains - some tokens are not saved in a separate log

### ISSUE THREAD

**Issue by Danimos552** [Bionus/imgbrd-grabber#1429]
**Bug description**

Some tokens are not saved to a separate log. These are %general%, %date% and %source%. This happens both during batch and manual (individual) download.

**Steps to reproduce**

1. Go to Tools --> Options --> Save --> Separate log files
2. Click on 'Add a separate log file' and fill it with desired tokens (%artist%, %date%, %source%, etc)
3. Save the options
4. Choose a booru subdomain from the sources - <some_booru>.booru.org
5. Click get this page
6. Go to Downloads tab and click Download button
7. Once the batch is finished, go to the save folder, open a log file
8. There are empty lines where there should be information from tokens

**Expected behavior**

The information should be there, but it's not.

**Context**

All 3 files in the attachment
[main.log](https://github.com/Bionus/imgbrd-grabber/files/2407144/main.log)
[restore.log](https://github.com/Bionus/imgbrd-grabber/files/2407145/restore.log)
[settings.log](https://github.com/Bionus/imgbrd-grabber/files/2407163/settings.log)
I changed extensions of restore.igl and setting.ini to log, cause GitHub nagged that it doesn't support these formats.

**System information**

- OS: Ubuntu 18.04 (64bit)
- Grabber version: 6.0.5

**Comment by Bionus** [comment]
I'll close this issue as a duplicate of #1428 since they're the same issue (just applied to different sources).
I'll comment on the other one too 😄
